src/tests/ftest/rebuild/with_ior.py

```diff
@@ -38,16 +38,7 @@ def test_rebuild_with_ior(self):
         ior_timeout = self.params.get("ior_timeout", '/run/ior/*')
         iorflags_write = self.params.get("write_flg", '/run/ior/iorflags/')
         iorflags_read = self.params.get("read_flg", '/run/ior/iorflags/')
-        dfuse_mount_dir = self.params.get("mount_dir", "/run/dfuse/*")
-        test_file = self.params.get("test_file", "/run/ior/*")
-        ior_api = self.params.get("ior_api", '/run/ior/*')
-        obj_class = self.params.get("obj_class", '/run/ior/*')
-        transfer_size = self.params.get("transfer_size",
-                                        "/run/ior/transfer_blk_size_rebld/*")
-        block_size = self.params.get("block_size",
-                                     "/run/ior/transfer_blk_size_rebld/*")
-        rank = self.params.get("rank_to_kill",
-                               "/run/ior/transfer_blk_size_rebld/*")
+        rank_to_kill = self.params.get("rank_to_kill", "/run/ior/*")
```

Comment:
Does this make sense as a `/run/ior/*` parameter or should it just be defined in `/run/*`?

Reply:
You're right. Other rebuild cases do something like:
```
testparams:
  ranks:
    rank_to_kill: 3
```
I'll fix this if there are other changes. Otherwise, probably better to not run through the (CI) grinder again.
FWIW, in a different PR I added a utility function `stop_random_rank`, which could be a simpler replacement for some cases like this.